api/python/notebooks/comp_bio_census_info.ipynb

```diff
@@ -1056,13 +1056,13 @@
        "3         CD8-positive, alpha-beta memory T cell             True\n",
        "4                           tracheal goblet cell             True\n",
        "...                                          ...              ...\n",
-       "33861071                         mesodermal cell             True\n",
-       "33861072                   colon epithelial cell             True\n",
-       "33861073                    enteroendocrine cell             True\n",
-       "33861074                   colon epithelial cell             True\n",
-       "33861075              enteric smooth muscle cell             True\n",
+       "41487772                         mesodermal cell             True\n",
+       "41487773                   colon epithelial cell             True\n",
+       "41487774                    enteroendocrine cell             True\n",
+       "41487775                   colon epithelial cell             True\n",
+       "41487776              enteric smooth muscle cell             True\n",
        "\n",
-       "[33861076 rows x 2 columns]"
+       "[41487777 rows x 2 columns]"
```

Comment:
Need to update the text below which now quotes the wrong number of cells.

Reply:
perhaps write in such a way that it does not go immediately obsolete when we re-run the notebook?  E.g.,  "The above row count is the number of cells ..."

Reply:
Fixed